### PR TITLE
chore: add DartPad Firestore example

### PR DESCRIPTION
**Pull request by Ehesp**
Adds `DartPad` components to MDX:

flutter-firebase-docs--pr7344-dartpad-qls1nlc2 web app_docs_firestore_example_

### CODE REVIEW

**Comment by Ehesp** [comment]
https://flutter-firebase-docs--pr7344-dartpad-qls1nlc2.web.app/docs/firestore/example

File: website/src/theme/MDXComponents/index.tsx [hunk 1]
```diff
@@ -80,6 +80,32 @@ export default {
...
+    return (
+      <div
+        style={{
+          marginBottom: '2rem',
+          position: 'relative',
+          paddingTop: '56.25%',
+        }}
+      >
+        <iframe
+          src={`https://dartpad.dev/embed-flutter.html?theme=dark&run=true&split=70&null_safety=true&gh_owner=FirebaseExtended&gh_repo=flutterfire&gh_ref=${
```
Comment (Salakar) [inline website/src/theme/MDXComponents/index.tsx:93]: Any way to set `theme=` here to match the value of the site theme toggle?

**Comment by Salakar** [comment]
Nice. I'd probably sleep easier if we used a new firebase project for the examples and avoid using `react-native-firebase-testing` before we make this live - just conscious of users hacking around with these credentials on DartPad and causing issues/rate limiting with e2e tests on RNFB & FF

File: website/src/theme/MDXComponents/index.tsx (hunk 1 above)
Comment (Ehesp) [reply website/src/theme/MDXComponents/index.tsx]: Yeah good idea, done.

**Comment by q464pp** [comment]
可以拉取合併嗎？謝謝

File: docs/firestore/example.mdx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: Firestore Example
+sidebar_label: Example
+hide_table_of_contents: true
+---
+
+<DartPad branch="dartpad" path="packages/cloud_firestore/cloud_firestore/example" />
```
Comment (Ehesp) [inline docs/firestore/example.mdx:7]: TODO: change to correct branch once merged / before merging

**Comment by q464pp** [comment]
可以拉取合併嗎 謝謝

**Pull request merged**